### DISCUSSION TITLE
Install packages using mamba

### DISCUSSION THREAD

**Discussion by FispraBappedaSultra** [giswqs/geog-312#42] [Q&A] [answered]
Hello Dr. Wu. I am following your course on GIS programming. I was following the instructions to Install packages using mamba: mamba install -c conda-forge geemap leafmap

but I keep getting this error message: 
![Screenshot 2024-09-04 164123](https://github.com/user-attachments/assets/c55361b3-1b48-44e5-a25a-667b680319fd)

**Comment by giswqs** [comment, marked as answer]
The timeout error indicates that it is your network problem. Try `pip install geemap leafmap` if conda does not work for you.

**Reply by FispraBappedaSultra** [reply]
thank you sir
